Lib/test/test_httpservers.py

```diff
@@ -46,13 +51,23 @@ def read(self, n=None):
 
 
 class TestServerThread(threading.Thread):
-    def __init__(self, test_object, request_handler):
+    def __init__(self, test_object, request_handler, tls=None):
```

Comment:
I was thinking of adding the `create_https_server` here, just before this class.

Reply:
I got you